### PR TITLE
moving to new CQ export method

### PR DESCRIPTION
**Pull request by Shimwell**
## Proposed changes

This removes the opening of a file and use of exporters.exportShape and replaces it with exporters.export as discussed in #82 

The former is now depreciated.

The new option will also support additional SVG options via an opt keyword once CQ 2.1 is released 

Also slightly more control over the stl export is also offered via the introduction of a new argument called ```angular_tolerance```

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Code refactoring
- [ ] Documentation Update (if none of the other choices apply)
- [ ] New tests

## Checklist

- [ ] Pep8 applied
- [ ] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: paramak/shape.py [hunk 1]
```diff
@@ -792,14 +796,10 @@ def export_stp(
...
-                exporters.exportShape(self.solid, "STEP", out_file)
-            elif mode == 'wire':
-                exporters.exportShape(self.wire, "STEP", out_file)
-            else:
-                raise ValueError("The mode argument for export_stp \
-                    only accepts 'solid' or 'wire'", self)
+        if mode == 'solid':
+            exporters.export(self.solid, str(path_filename), exportType='STEP')
+        elif mode == 'wire':
+            exporters.export(self.wire, str(path_filename), exportType='STEP')
```
Comment (Shimwell) [inline paramak/shape.py:802]:
```suggestion
            exporters.export(self.wire, str(path_filename), exportType='STEP')
        else:
            raise ValueError("The mode argument for export_stp \
                only accepts 'solid' or 'wire'", self)
```

**Review by billingsley-john — APPROVED** [review]
This looks like a nice addition to make things more tidy!

**Comment by Shimwell** [comment]
Not sure why this is failing, I've tried switching back to the original stl export and changing pymoab to the main branch but the github actions CI are still failing

**Comment by Shimwell** [comment]
I need to redo parts of this PR as #718 has updated the svg export

**Comment by Shimwell** [comment]
closing this a #720 #718 and #721 have added the features

**Pull request closed without merging**